.github/workflows/release-please.yml

```diff
@@ -0,0 +1,40 @@
+###############################################################
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+name: Release Please
+
+on:
+  push:
+    branches:
+      - 'v*.*.*'
+  workflow_dispatch:
+
+permissions:
+    contents: write
+    pull-requests: write
+
+jobs:
+    prepare-release:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: google-github-actions/release-please-action@v4
+          name: Prepare release
+          with:
+            target-branch: ${{ github.ref_name }}
+            release-type: simple
```

Comment:
could you please add the skip-release option? https://github.com/eclipse-tractusx/policy-hub/blob/main/.github/workflows/release-please.yml#L41

Reply:
done